src/table/cells/handle-key-press.js

```diff
@@ -18,17 +18,18 @@ export const updatePage = (evt, totalRowSize, page, rowsPerPage, handleChangePag
 
 export const arrowKeysNavigation = (evt, rowAndColumnCount, cellCoord, selState) => {
   let [nextRow, nextCol] = cellCoord;
-  const isSelectedTable = selState && selState.rows.length > 0;
+  // check if you have unconfirmed selections, so one or more cells are selected but not confirmed yet.
+  const isUnconfirmedSelection = selState && selState.rows.length > 0;
```

Comment:
I think the correct terminology is `hasUnconfirmedSelection`

Reply:
Better! Thanks. updated.